book.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+let myLibrary = [];
+
+function Book(author, title, pageCount, hasRead) {
+  const book = {
+    author,
+    title,
+    pageCount,
+    hasRead,
+  };
+  return book;
+}
+
+function clearDisplay() {
+  const bookholder = document.getElementById('bookholder');
+  bookholder.innerHTML = '';
+}
+
+function displayLibrary() {
+  clearDisplay();
+}
+
+function addBookToLibrary() {
+  const author = document.getElementById('author').value;
+  const title = document.getElementById('title').value;
+  const pageCount = document.getElementById('pageCount').value;
+  const hasRead = document.getElementById('check').checked;
+  const newBook = Book(author, title, pageCount, hasRead);
+  myLibrary.push(newBook);
+  displayLibrary();
+
+  const savedLibrary = JSON.stringify(myLibrary);
+  localStorage.setItem('book', savedLibrary);
+  // const book =
```

Comment:
- [ ] <b>OPTIONAL:</b> Keep your code clean by remembering to always remove or delete `commented-out codes`.

Reply:
Gotcha